delphi/translators/for2py/pyTranslate.py

```diff
@@ -181,11 +181,13 @@ def __init__(self):
         # names
         self.nameMapper = {}
         # Dictionary to hold functions and its arguments
+        self.functions = []
```

Comment:
Please add a comment to each of these fields saying what it's for.  Also, please elaborate on the comment before `nameMapper`, which currently is not very helpful.